verification/benchmark_user_events_v2.php

```
@@ -0,0 +1,194 @@
+    public function get_quantity() { return $this->quantity; }
+}
+
+class MockOrder {
+    public $items = [];
+    public function __construct($items) {
+        $this->items = $items;
+    }
+    public function get_items() { return $this->items; }
+}
+
+function wc_get_orders($args) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$args'. (undefined)

(UnusedFormalParameter)